### PR TITLE
[fix] Make can output of `Multiple possible types`

### PR DESCRIPTION
**Pull request by ota-meshi**
input:
```vue
  props: {
    propB: [String, Number],
  }
```

output:
```md
## props 
- `propB` ***String|Number***
```

reference:
https://vuejs.org/v2/guide/components-props.html#Prop-Validation
`propB `

### CODE REVIEW

File: lib/markdown.js [hunk 1]
```diff
@@ -56,6 +56,13 @@ const parseValue = (value) => {
   return value
 }
 
+const parseType = (type) => {
+  if (type && type.type === 'ArrayExpression') {
+    return type.elements.map((e) => e.name).join('|')
```
Comment (demsking) [inline lib/markdown.js]: What is the meaning of the `e` variable?

Comment (demsking) [reply lib/markdown.js]: Could you provide a more meaningful variable name please?

**Review by demsking — CHANGES_REQUESTED** [review]
Thank you for this PR :-)

**Comment by ota-meshi** [comment]
@demsking Thank you for review!
I changed the variable name, as your requested.
Please check again it.

**Review by demsking — APPROVED** [review]

**Comment by demsking** [comment]
Merged to the `develop` branch. I'll release a new version soon

**Comment by demsking** [comment]
Released: [@vuedoc/md@1.2.0](https://github.com/vuedoc/md/releases/tag/v1.2.0)

**Pull request merged**